p/themes/Offerel/metadata.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "Offerel",
+  "author": "Offerel",
+  "description": "A Fork of Swage.",
+  "version": 0.9,
+  "files": ["_template.css", "swage.css", "offerel.css"]
```

If possible, just link to the base Swage theme this way (not tested):

```suggestion
  "files": ["_template.css", "../Swage/swage.css", "offerel.css"]
```

---

I have changed your suggestion, which worked and also removed the no un-needed files from my theme. I got a (automated?) mail, that there was Push initiated. Maybe this was something automated, because i committed the changes to my branch.